test/util.jl

```diff
@@ -31,4 +31,12 @@ import SPICE
     @testset "plane_section" begin
         xy = plane_section([1.0, 2.0, 3.0], zeros(3), [0.0, 0.0, 1.0])
     end
+    @testset "spherical_to_cartesian" begin
+        act = spherical_to_cartesian(3.0123, -0.999)
+        exp = (-0.5366267667260523906, 0.0697711109765145365, -0.8409302618566214041)
+        @testset for i in eachindex(exp, act)
+            @test isapprox(exp[i], act[i], rtol=0, atol=1e-12)
```

Comment:
```suggestion
            @test exp[i] ≈ act[i] atol=1e-12
```

Reply:
You can omit `rtol` if you provide `atol`

Reply:
Speaking for myself, usually I prefer `rtol` to `atol`, an absolute error tells nothing about the significance of an error.

Reply:
Unless the value is close to zero in which case you should definitely use `atol` 😉

Reply:
Fair enoguh :slightly_smiling_face: